modules/virt-enabling-disabling-vm-delete-protection-web.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-enabling-disabling-vm-delete-protection-web_{context}"]
+
+= Enabling or disabling virtual machine delete protection by using the web console
+
+To prevent the inadvertent deletion of a virtual machine (VM), you can enable virtual machine delete protection by using the {product-title} web console. Should you change your mind, you can easily disable delete protection for a VM.
```

Comment:
Some of the same comments in the above module apply here, if you can check those